.pre-commit-config.yaml

```diff
@@ -60,7 +60,13 @@ repos:
       - id: gitlint
         stages: [commit-msg]
         args: [--contrib=CT1, --msg-filename]
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: "v4.0.0-alpha.8"
+  - repo: local
     hooks:
-      - id: prettier
+      - id: htmlformatter
+        name: html prettier formatter
+        language: node
+        additional_dependencies:
+          - prettier@3.3.2
+          - prettier-plugin-jinja-template@1.4.0
+        entry: npx prettier task_manager --write **/*.html
+        types_or: [html]
```

Comment:
let's move all `local` repo hooks into one place - we have `mypy` hook defined in the `local` repo as well